nautobot_ssot_infoblox/diffsync/models/nautobot.py

```diff
@@ -168,7 +166,7 @@ def create(cls, diffsync, ids, attrs):
             rir=rir,
             description=attrs["description"] if attrs.get("description") else "",
         )
-        _aggregate.tags.add(create_tag_sync_from_infoblox())
+        _aggregate.tags.add("SSoT Synced to Infoblox")
```

Comment:
I learned something new,  I thought this required an object.

Reply:
This was blowing up complaining about the Tag object being passed and insisted on a string. This might be a change in the Nautobot version since this was originally using 1.0.1 I believe?

Reply:
I guess the question is what's the minimum version of Nautobot we want to set? I feel like 1.3 is reasonable and has some features we might want to use in this plug-in.